src/tools/print-schema/print-schema.js

```diff
@@ -0,0 +1,21 @@
+/**
+ *  Copyright (c) 2015, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+require('babel/register')({
+  optional: ['runtime', 'es7.asyncFunctions']
+});
+
+var executeTool = require('./src').executeTool;
+
+executeTool().then(function (out) {
```

Comment:
The separation between script and src feels odd to me; this does the `stdout` stuff but the src does the file stuff? I almost wish this was just one script, since the "guts" of the script are only three lines, everything else is managing params and output.